_posts/2024-01-10-15.0.0-release.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: "Apache Arrow 15.0.0 Release"
+date: "2024-01-10 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 15.0.0 release. This covers
+over 3 months of development work and includes [**XXX resolved issues**][1]
+from [**YYY distinct contributors**][2]. See the [Install Page](https://arrow.apache.org/install/)
+to learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 14.0.0 release, Curt Hagenlocher, Xuwei Fu, James Duong and Felipe Oliveira Carvalho
+have been invited to be committers.
+Jonathan Keane and Raúl Cumplido have joined the Project Management Committee (PMC).
+
+As per our tradition of rotating the PMC chair once a year
+Andy Grove was elected as the new PMC chair and VP.
+
+Thanks for your contributions and participation in the project!
+
+## Columnar Format Notes
+
+## C Data Interface notes
+
+## Arrow Flight RPC notes
+
+## C++ notes
+
+### Compute layer
+
+### Datasets
+
+### Gandiva
+
+### Parquet
+
+### Substrait
+
+### Miscellaneous
+
+## C# notes
+
+## Go Notes
+
+### Bug Fixes
+
+### Enhancements
+
+## Java notes
+
+## JavaScript notes
+
+## Python notes
+
+## R notes
```

Comment:
@paleolimbot @assignUser @thisisnic could you help  with the R notes?

Reply:
```suggestion
## R notes

### New features:

* Bindings for `base::prod` have been added so you can now use it in your dplyr pipelines (i.e., `tbl |> summarize(prod(col))`) without having to pull the data into R [GH-38601](https://github.com/apache/arrow/pull/38601).
* Calling `dimnames` or `colnames` on `Dataset` objects now returns a useful result rather than just `NULL` [GH-38377](https://github.com/apache/arrow/pull/38377).
* The `code()` method on Schema objects now takes an optional `namespace` argument which, when `TRUE`, prefixes names with `arrow::` which makes the output more portable [GH-38144](https://github.com/apache/arrow/pull/38144).

### Other improvements:

* To make debugging problems easier when using arrow with AWS S3 (e..g, `s3_bucket`, `S3FileSystem`), the debug log level for S3 can be set with the `AWS_S3_LOG_LEVEL` environment variable. See `?S3FileSystem` for more information. [GH-38267](https://github.com/apache/arrow/pull/38267)
* An error is now thrown instead of warning and pulling the data into R when any of `sub`, `gsub`, `stringr::str_replace`, `stringr::str_replace_all` are passed a length > 1 vector of values in `pattern` [GH-39219](https://github.com/apache/arrow/pull/39219).
* Missing documentation was added to `?open_dataset` documenting how to use the ND-JSON support added in arrow 13.0.0 [GH-38258](https://github.com/apache/arrow/pull/38258).
* Using arrow with duckdb (i.e., `to_duckdb()`) no longer results in warnings when quitting your R session. [GH-38495](https://github.com/apache/arrow/pull/38495)
```
@amoeba I've just copy&pasted here your comment as a commit suggestion. I'll merge it.